docs/source/guides/understanding_types_and_tags.ipynb

```diff
@@ -476,7 +476,109 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "After calling `set_time_index`, the `time_index` semantic tag has been added to the semantic tags for `times` column."
+    "After calling `set_time_index`, the `time_index` semantic tag has been added to the semantic tags for `times` column.\n",
+    "\n",
+    "## Validating Woodwork's Typing Information\n",
+    "\n",
+    "The logical types, physical types, and semantic tags described above make up a DataFrame's typing information, which will be referred to as its \"schema\". For Woodowork to be useful, the schema must be valid with respect to its DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.ww.schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Woodwork schema shown above can be seen reflected in the DataFrame below. Every column present in the schema is present in the DataFrame, the dtypes all match the physical types defined for each column, and the index column is both unique and represented on the DataFrame's underlying index."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.dtypes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The question of what a valid schema is for a DataFrame is defined by Woodwork, and it is reliant on the DataFrame following Woodwork's type system. For this reason, it is not recommended to perform DataFrame operations directly on the DataFrame; instead, you should go through the `ww` namespace. If a DataFrame operation invalidates the Woodwork schema defined for that DataFrame, the typing information will be removed.\n",
```

Comment:
How about "Woodwork defines the elements of a valid schema, and maintaining schema validity requires that the DataFrame follow Woodwork's type system." for the first sentence?

Also, maybe we could add a sentence prior to the last sentence to reinforce using the `ww` namespace: "Woodwork will attempt to retain a valid schema for any operations performed through the `ww` namespace."

Reply:
That first sentence is much more clear, thanks!